contribute/quality-assurance.rst

```diff
@@ -1,7 +1,9 @@
 Quality Assurance
 =================
 
-This page explains how to help the UBports QA team, both as an official member or a new contributor. Please also read the :doc:`issue tracking </about/process/issue-tracking>` and :doc:`bugreporting <bugreporting>` guides to better understand the workflow. For real-time communication, you can join our `telegram group <http://t.me/ubports_qa_team>`_.
+This page explains how to help the UBports QA team, both as an official member or a new contributor.
+Please also read the :ref:`bug tracking <bug-tracking>` and :ref:`bug reporting <bug-reporting>` guides to better understand the workflow.
```

Comment:
bug-tracking here then needs to be issue management if you follow my suggestion above

Reply:
well, I also like the pattern/repetition with bug reporting / bug triaging. we could change *both* but rather not only one of them

Reply:
I guess it works out well as it is now. It would have only made sense when doing that other renaming. Not needed now. All good.